### PR TITLE
Added test worker and deleting pods, volumes after build is complete

### PR DESCRIPTION
**Pull request by bamachrn**

### CODE REVIEW

File: server/bootup_openshift.sh [hunk 1]
```diff
@@ -54,7 +54,13 @@ dirs=(openshift.local.volumes openshift.local.config openshift.local.etcd)
 for d in ${dirs[@]}; do
   mkdir -p ${ORIGIN_DIR}/${d} && chcon -Rt svirt_sandbox_file_t ${ORIGIN_DIR}/${d}
 done
-
+################New Run command###############
+sudo docker run -d --name "origin" \
+        --privileged --pid=host --net=host \
+        -v /:/rootfs:ro -v /var/run:/var/run:rw -v /sys:/sys -v /var/lib/docker:/var/lib/docker:rw \
+        -v /var/lib/origin/openshift.local.volumes:/var/lib/origin/openshift.local.volumes:rslave \
```
Comment (rtnpro) [inline server/bootup_openshift.sh]: Why use ``rslave`` when mounting volume?

File: provisions/roles/openshift/defaults/main.yml [hunk 2]
```diff
@@ -17,9 +17,9 @@ openshift_shared_dirs:
 openshift_volumes:
   - /:/rootfs:ro
   - /var/run:/var/run:rw
-  - /sys:/sys:ro
+  - /sys:/sys
   - /var/lib/docker:/var/lib/docker:rw
-  - /var/lib/origin/openshift.local.volumes:/var/lib/origin/openshift.local.volumes:rw
+  - /var/lib/origin/openshift.local.volumes:/var/lib/origin/openshift.local.volumes:rslave
```
Comment (rtnpro) [inline provisions/roles/openshift/defaults/main.yml]: Why use ``rslave``? Ansible does not support it yet.

Comment (bamachrn) [inline provisions/roles/openshift/defaults/main.yml]: rslave is mandatory for v1.4.1

**Comment by rtnpro** [comment]
#dotests-debug 7200

**Comment by rtnpro** [comment]
#dotests-debug 7200

**Comment by rtnpro** [comment]
#dotests-debug

**Comment by rtnpro** [comment]
#dotests-debug

**Comment by rtnpro** [comment]
#dotests

**Comment by rtnpro** [comment]
#dotests-debug

**Pull request merged**